.changeset/dirty-cars-work.md

```diff
@@ -0,0 +1,9 @@
+---
+"apollo-client-devtools": patch
```

Comment:
Minor? Not sure how best to classify this since its more or less a giant refactor. Perhaps the new message format classifies it as a minor?